R/mod_metric_details_server.R

```diff
@@ -88,7 +88,6 @@ mod_metric_details_server <- function(
           output$results <- renderUI({
             gsm::Report_MetricTable(
               rctv_dfResults_byMetricID() %>%
-                filter_byGroupID(rctv_strSite()) %>%
```

Comment:
Do we not want Analysis Output to respect the Site filter? It isn't clear to me why this was removed.

Reply:
The issue here is the analysis output is the subset of sites with a non-zero flag, which doesn't necessarily include the selected site.

Reply:
Yeah, I get it after digging in a bit more. I'm writing a ticket to make this more interactive; like the charts, I feel like it should *show* everything, but allow the user to select a site (and update the associated drop-down), and it should highlight the selected site when a site is selected.

Reply:
Logged in #176.